vulnerabilities/tests/test_data/oss_fuzz/oss-fuzz-expected1.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Please rename these expected files to use the same base name as a the test data file with an `-expected.json` suffix. Here do not use `oss-fuzz-expected1.json`. Instead use `oss-fuzz-data1.yaml-expected.json` .... ths way the test data file and the expected results show up side by side.

Reply:
Done!